src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+
+interface Props {
+  error: string,
+  setHideError: (value: boolean) => void,
+  hideError: boolean,
+  setError: (value: string) => void
+}
+
+export const ErrorMessage: React.FC<Props> = ({
+  error,
+  setHideError,
+  hideError,
+  setError,
+}) => {
+  setTimeout(() => setError(''), 3000);
```

Comment:
better use useEffect, you need to create setTimeout on the first render and clear timeout on unmount